keyboards/massdrop/thekey/readme.md

```diff
@@ -0,0 +1,36 @@
+# The Key
+
+![The Key](https://i.imgur.com/hL5cRj9.jpg)
+
+
+The Stack Overflow "The Key" is a 3 button macropad based on atmega32u4 with Kailh Black Box switches.
+
+> Some say a programmer’s best solution is a simple two-step process: copy and paste. On April 1st, 2021, Stack Overflow proved it. On that fateful day, each time users went to copy a piece of code, they were met with a pop-up for a fake product. It was called The Key: an ultra-compact macropad advertised as the new (and only) way to copy and paste on the platform. As it turns out, roughly one fourth of Stack Overflow’s 15 million users tries to copy and paste within five minutes of visiting the site. An even greater number saw the joke and loved it, with many demanding that The Key be developed in earnest. So, naturally, we teamed up with Stack Overflow to make it happen. What started as an April Fool’s gag is now a full-fledged macropad—designed by our very own community member Cassidy, with a portion of proceeds benefiting digitalundivided.
+
+Keyboard Maintainer: [Drop / Massdrop](https://github.com/Massdrop/qmk_firmware) 
+
+Hardware Supported: Massdrop, Inc. **The Key**
+
+Hardware Availability: Limited Release - https://drop.com/buy/stack-overflow-the-key-macropad
+
+Make example for this keyboard (after setting up your build environment):
+
+```bash
+# build 
+make massdrop/thekey:default
+# install in dfu mode
+make massdrop/thekey:default:dfu
+```
+## Bootloader
+
+Enter the bootloader as follows:
+* **Physical reset button**: Briefly press and hold the reset button while pluggin in the USB port. The LEDs on the back will **NOT** turn on. Depending on your case revision, you may have to remove the 4 screws on the back plate to access the switch OR you can use the associated access hole on newer releases.
+
+## Additional Details
+For information on flashing this keyboard, visit the following links:
+
+[Massdrop Loader Releases](https://github.com/Massdrop/mdloader/releases/tag/1.0.3)  
+[Massdrop Loader Repository and Instructions](https://github.com/Massdrop/mdloader)[Massdrop Loader Releases]
+[Massdrop Loader Repository and Instructions]
```

Comment:
`mdloader` is for their ATSAMD-based boards, i.e. their CTRL/ALT etc. -- it doesn't function with AVR 32U4-based boards.
You'll want to grab the same wording from another 32U4-based board in the repo, here.

Reply:
yea, this was my bad trying to fix something while someone else was talking to me. they don't actually have this information on their repos, just buried in the forums. it's probably best to remove it and just leave the verified instructions i have above.